tests/conftest.py

```diff
@@ -40,6 +40,41 @@ def make_flow(port_generator, request):
         yield f
 
 
+@pytest.fixture(
+    scope='session',
+    params=[
+        ['onnx', 'ViT-B-32::openai'],
+        ['onnx', 'ViT-H-14::laion2b-s32b-b79k'],
+        ['torch', 'ViT-B-32::openai'],
+        ['onnx_custom', 'ViT-B-32::openai'],
+    ],
+)
+def make_flow_with_large(port_generator, request):
```

Comment:
Consider move this test to test_model and only test for onnx + fp16 + ViT-H-14::laion2b-s32b-b79k